### PR TITLE
Added fix for week number offset

### LINKED ISSUE #44: incorrect menu for fmi-bistro
I've noticed that the API returns the wrong dishes for fmi-bistro. The PDFs published [here](https://www.wilhelm-gastronomie.de/tum-garching) show the current menu.

For instance today (it's Friday 11.01.2019) the API returns:

Minestrone | 1.00€
Grieß-Kokosmilch mit warmen Himbeeren | 3.20€
Schwäbischer Ofenschlüfer | 3.20€
Fischfilet in Kartoffelpanade dazu Kartoffel-Fenchelgemüse | 4.60€

But the actual menu is as follows:

Menu Friday 11.01.2019

### PR DESCRIPTION
**Pull request by hbrgnr**
* Added some basic contributing instructions
* Switched to %G for years and %V for weeks since in Germany we use ISO 8601 for week numbering
* Added a test for the date parsing

Fixes #44

### CODE REVIEW

**Comment by srehwald** [comment]
Thanks a lot! Will try to merge it soon!

File: README.md [hunk 1]
```diff
@@ -96,3 +96,13 @@ $ python src/main.py mensa-arcisstrasse -d 02.04.2017
...
 - UWP-TUM-Campus-App ([Github](https://github.com/COM8/UWP-TUM-Campus-App))
+
+## Contributing
+
+1. Fork and clone this repository
+2. Install the python dependencies:
+  * `sudo apt install libxml2 libxml2-dev libxslt1-dev`
+  * `pip3 install -r requirements.txt`
+3. Run tests:
+  * All the test: `PYTHONPATH=src/ python3 -m unittest`
```
[the remark this answers is not in the document]
Comment (hbrgnr) [reply README.md]: thanks, pytest is indeed much nicer. is there an alternative to `PYTHONPATH=src/ python3 -m unittest test.test_menu_parser.MenuParserTest` in pytest syntax? couldn't figure it out

File: src/menu_parser.py [hunk 2]
```diff
@@ -21,8 +21,12 @@ class MenuParser:
     def get_date(self, year, week_number, day):
         # get date from year, week number and current weekday
         # https://stackoverflow.com/questions/17087314/get-date-from-week-number
-        date_str = "%d-W%d-%d" % (year, week_number, day)
-        date = datetime.strptime(date_str, "%Y-W%W-%w").date()
+        # but use the %G for year and %V for the week since in Germany we use ISO 8601 for week numbering
+        date_format = "%G-W%V-%w"
```
[the remark this answers is not in the document]
Comment (hbrgnr) [reply src/menu_parser.py]: thanks, totally missed that

**Comment by hbrgnr** [comment]
Regarding the python3.6: the Dockerfile seems to be using python3.6 but `scripts/parse.sh` uses `python`. Not sure where that's in use? Would we be safe if we changed it to `python3`?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (raabf) [reply README.md]: pytest can take the filename as arguments, class names can be appended, example:

`PYTHONPATH=$(pwd)/src pytest src/test/test_menu_parser.py::StudentenwerkMenuParserTest`

Alternative (-k searches the string inside the "…"):

`PYTHONPATH=$(pwd)/src pytest -k "StudentenwerkMenuParserTest"`

File: README.md [hunk 3]
```diff
@@ -96,3 +96,13 @@ $ python src/main.py mensa-arcisstrasse -d 02.04.2017
...
+
+## Contributing
+
+1. Fork and clone this repository
+2. Install the python dependencies:
+  * `sudo apt install libxml2 libxml2-dev libxslt1-dev`
+  * `pip3 install -r requirements.txt`
+3. Run tests:
+  * All the tests: `PYTHONPATH=src/ pytest`
+  * A specific test file: `PYTHONPATH=src/ pytest src/test/test_menu_parser.py`
```
Comment (hbrgnr) [inline README.md]:
```suggestion
  * A specific test class: `PYTHONPATH=src/ pytest src/test/test_menu_parser.py::MenuParserTest` 
```

**Comment by raabf** [comment]
@hbrgnr Depends on the deployment, when an virtualenv is activatet to a python 3 version, both commands `python` and `python3` are set to the interpreter of the virtualenv, so it does not matter. When it is deactivated `python` normally points to the system python 2 version, so using `python3` would be a litte be safer, but I think @srehwald must decide that, since he uses the scripts for deployment.

File: README.md (hunk 3 above)
Comment (raabf) [reply README.md]: should work, yes!

**Comment by srehwald** [comment]
Will be merged, finally. Thanks!

Regarding the python version: The main reason I am using `python` in the scripts was that I am explicitly setting it to version 3.6 in `.travis.yml`. I guess I should mention in the readme that Python 3.6 is the version you need to get the code running.

**Pull request merged**